nextflow_schema.json

```diff
@@ -376,6 +376,11 @@
                     "type": "boolean",
                     "description": "Skip read preprocessing using fastp or adapterremoval."
                 },
+                "skip_shortread_qc": {
+                    "type": "boolean",
+                    "description": "Skip all QC steps for short reads.",
+                    "help": "Skips clipping and removal of phiX sequences as well as the second FastQC run. Does not affect host sequence removal."
```

Comment:
Eyeballing the code - the second FastQC run should still run if host removal is enabled?

Reply:
Good point, didn't think of that.